site.maintenance.aws-remove-all.yml

```diff
@@ -144,6 +151,8 @@
             eni_id: "{{ item }}"
             force_detach: true
           with_items: "{{ _aws_eni_to_remove }}"
+          ignore_errors: true
```

Comment:
we should not ignore errors here,
this is failing because aws does not allow manually managing network interfaces for lambdas
https://aws.amazon.com/premiumsupport/knowledge-center/lambda-eni-find-delete/
This step should be removed as is not required anymore